cmd/kube-apiserver/apiserver.go

```diff
@@ -56,6 +56,9 @@ var (
 		"and it is assumed that port 443 at this address will be proxied/redirected "+
 		"to '-address':'-port'. If blank, the address in the first listed interface "+
 		"will be used.")
+	internalServiceAddress = flag.String("internal_service_address", "10.0.0.20", ""+
```

Comment:
The problem is that portal_net is necessarily configurable.  This means you just CAN'T allocate a static IP like this.  It either has to come from user config (e.g. config-default.sh) or it has to be carried relative to portal_net.  Why not just assume that the master uses portal_net + 1 - you can even use some of the IP-math logic in ip_allocator.go if you export it properly.